### ISSUE TITLE
Preserve attributes of elements.

### ISSUE THREAD

**Issue by songww** [carrascomj/bevy_svg_map#10]
Such as `id` and `class`.

**Comment by carrascomj** [comment]
I can see how that would be useful. What kind of API do you propose for that?

I would think of something like parsing and adding `id` and `class` for the `SvgStyle` (although it is not part of style, but whatever) and then the user could add to the component like

```rust
/// User defined component
struct Svg {
    id: String,
    class: String,
}

/// User defined strategy
struct MyStrategy;

impl StyleStrategy for MyStrategy {
    fn component_decider(&self, style: &SvgStyle, comp: &mut Commands) {
        // append a Svg with the id and class of the SVG path
        comp.with(
            Svg { id: style.id, class: style.class }
        );
    }
}
```

How does that look for you?

**Comment by songww** [comment]
looks good. thank you.

**Comment by carrascomj** [comment]
Closed at a1d44fe379ca20f38292fcb812ad80e9d7390818 (available at 0.2.0 once I release it).
